en/upgrades/decimal-upgrade-to-2-03.rst

```diff
@@ -4,3 +4,55 @@ Decimal Upgrade to 2.03
 The next decimal upgrade of the IATI Standard (to version 2.03) will take place from Friday 12 May through to early October 2017. The full timetable below outlines each phase of the upgrade process and  includes information on when input is invited from the IATI Community. We anticipate the end of this process and the release of version 2.03 on **Tuesday 3 October 2017**.
 
 Specific proposals and public consultations in the upgrade process will be available in the `2.03 section on IATI’s online forum <https://discuss.iatistandard.org/c/standard-management/2-03-decimal-upgrade-proposals>`__ from the afternoon of Friday 12 May – you can read, comment or add a new proposal there.
+
+2.03 Upgrade Timetable
+----------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Phase
+     - Stage Start Date
+     - Actions by/for the IATI tech team
+     - Actions by the Community
+     - Stage End Date
+   * - Suggestions
+     - Friday 12 May
+     - Announcement of 2.03 upgrade process, and request for any more new proposals is advertised to the Community.
+     - -
+     - Friday 12 May
+   * - Suggestions
+     - (afternoon of) Friday 12 May
+     - -
+     - The Community submits new and/or discusses existing proposals and ideas.
+     - Friday 26 May
+   * - Suggestions
+     - Saturday 27 May
+     - The technical team review and agree which proposals should be considered for the upgrade. These are shared with the Community.
+     - Proposals going forward are flagged as being ‘under consideration’. The Community are invited to comment on these and may be contacted for more information on their views.
+     - Friday 9 June
+   * - Consultation
+     - Saturday 10 June
+     - The technical team undergoes consultation with the Community.
+     - The Community can help refine the proposals and discuss in detail what has been included or left out.
+     - Friday 7 July
+   * - Consultation
+     - Saturday 8 July
+     - Proposals are refined and worked up into finished examples.
+     - The Community are still able to submit minor suggestions/changes, although this phase is mainly for the tech team to work up proposals into examples.
+     - Friday 28 July
+   * - Development and Inspection
+     - Saturday 29 July
+     - The technical team will develop proposals and add them to a dedicated development branch of the schema (on GitHub).
+     - The Community inspects and comments on the 2.03 changes on GitHub as they are being developed.
+     - Friday 15 September
+   * - Approval and Go Live
+     - Saturday 16 September
+     - Final version of 2.03 completed. We will work to ensure core IATI secretariat tools are compatible with the new version.
+     - -
+     - Monday 2 October
+   * - Approval and Go Live
+     - Tuesday 3 October
+     - Version 2.03 goes live.
+     - -
+     - Tuesday 3 October
```

Comment:
I am happy with this as long as Matt is happy.

(Why is Tuesday 3 October mentioned twice?)